### PR TITLE
Peer Review

### PR DESCRIPTION
**Pull request by TristanBarlow**
The file to be reviews is called "master.py" and is found in the "final" folder

### CODE REVIEW

File: final/master.py [hunk 1]
```diff
@@ -0,0 +1,405 @@
+import pygame, sys, time, random, math
+from pygame.locals import *
```
Comment (RobertsOz) [inline final/master.py]: Test

File: final/master.py [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+        clamped_value = 255
+    elif value < 0:
+        clamped_value = 0
+    else:
+        clamped_value = value
+    return clamped_value
+
+
+# changes colours if within the a certain range.
+def cap_colours(color_change):
```
Comment (RobertsOz) [inline final/master.py]: Me likey

File: final/master.py [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+            # Essentially this divides by a number, truncates the decimal portion then multiplies it back to scale
+            # for instance were the strength = 100, a value of 234 would go to 2.34 to 2 then return 200
+            new_red = clamp(simplify_strength * int(red / simplify_strength) + (simplify_strength / 2))
+            new_green = clamp(simplify_strength * int(green / simplify_strength) + (simplify_strength / 2))
+            new_blue = clamp(simplify_strength * int(blue / simplify_strength) + (simplify_strength / 2))
+
+            px_array[x, y] = (new_red, new_green, new_blue)
+
+
+def pixelise_is_a_word_stop_underlining_it_in_grey():
```
Comment (RobertsOz) [inline final/master.py:283]: what is pixelise, is that a typo?

**Review by RobertsOz — COMMENTED** [review]
Professional proficiency right here!

File: final/master.py [hunk 4]
```diff
@@ -0,0 +1,405 @@
+import pygame, sys, time, random, math
```
Comment (jack-maber) [inline final/master.py:1]: Should import packages on seperate lines according to PEP-8.

File: final/master.py [hunk 5]
```diff
@@ -0,0 +1,405 @@
...
+GREEN = (0, 255, 0)
+BLUE = (0, 0, 255)
+
+WIDTH = 900
+HEIGHT = 600
+
+window = pygame.display.set_mode((WIDTH, HEIGHT))
+
+# Insert picture name to load below
+picture = pygame.image.load('cool_cat.jpg')
```
Comment (jack-maber) [inline final/master.py:20]: Could load images in a loop if they were numbered I.E. 0.jpg

**Pull request closed without merging**